File: Arbitration/MPArbitration/ClientApp/src/app/model/authority-fee.ts
import { BaseFee } from "./base-fee";

export class AuthorityFee extends BaseFee {
    public authorityId = 0;

    constructor(obj?:any){
        super(obj);
        if(!obj)
            return;

        if(!!obj.authorityId)
            this.authorityId = obj.authorityId+0;
    }
}

File: Arbitration/MPArbitration/Model/AuthorityFee.cs
using System.Text.Json.Serialization;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;
using System.ComponentModel.DataAnnotations;

namespace MPArbitration.Model
{
    /// <summary>
    /// A configured fee for an Authority. All configured fees are copied into each new dispute.
    /// </summary>
    [Index(nameof(AuthorityId), IsUnique = false)]
    public class AuthorityFee : BaseFee
    {
        [JsonPropertyName("authorityId")]
        [Required]
        public int AuthorityId { get; set; } = 0;

    }
}
